src/components/font-awesome-icon/index.js

```diff
@@ -152,7 +160,10 @@ FontAwesomeIcon.Content = props => {
 	const prefix = props.value ? props.value.replace( /-.*$/, '' ) : props.prefix
 	const iconName = props.value ? props.value.replace( /^.*?-/, '' ) : props.iconName
 
-	const iconHTML = faGetSVGIcon( prefix, iconName )
+	let iconHTML
+	faGetIcon( prefix, iconName ).then( svg => {
+		iconHTML = svg
+	} )
```

Comment:
We might run into problems here because `feGetIcon` is a promise that would need to be resolved before getting the value. Since this is called in `.Content` which is called during the saving process, we cannot have any promises here because we aren't assured that we will be able to wait for this value to resolve.

You will need to change this into a synchronous way to get the icon
```suggestion
	faGetIcon( prefix, iconName ).then( svg => {
		iconHTML = svg
	} )
```